syntax = "proto3";

package inventory;

import "type/uuid.proto";
import "google/protobuf/timestamp.proto";

message BoundRequest {
    uuid id = 1;
    string type = 2;
    string description = 3;
    string status = 4;
    google.protobuf.Timestamp created_at = 5;
	google.protobuf.Timestamp updated_at = 6;
}

message BoundResponse {
    uuid id = 1;
    string type = 2;
    string description = 3;
    string status = 4;
    google.protobuf.Timestamp created_at = 5;
	google.protobuf.Timestamp updated_at = 6;
}

service Bound {
    rpc GetBound(BoundRequest) returns (BoundResponse) {};
    rpc GetBounds(BoundRequest) returns (stream BoundResponse) {};
}